### PR TITLE
Add Create Repo Event, PR Review Event and PR Review Comment Event and Add separate module for each event

### PR DESCRIPTION
**Pull request by Andre601**
## Checks

I have...

- [x] read and understood the [Contributing Guidelines][contributing]
- [x] Updated any nessecary files such as the README.md and/or CHANGELOG.md.

## Type of Pull request

- [x] **Minor Change**  
  This Pull request doesn't break existing configuration.
- [ ] **Major Change**  
  This Pull request will break existing configuration.
- [ ] **Bug fix**  
  This Pull request will fix a (critical) bug.
- [ ] **Documentation**  
  This Pull request only changes documentation (README.md, CHANGELOG.md, etc.)
- [ ] **Other:** `__________` 

## Description

Adds new options for displaying actions for creating a repository, commenting on a Commit or Pull request (Covered by the `COMMENTS` option) or Reviewing a Pull request.

This was made by @abhijoshi2k. I only made this draft PR to update the changelog :P

Closes #

[contributing]: https://github.com/Readme-Workflows/recent-activity/blob/main/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by PuneetGopinath** [comment]
@abhijoshi2k What is the `Commit Comment Event` in the title?
I think the title should be changed to `Add Create Repo Event, PR Review Event and PR Review Comment Event and Add separate module for each event`

File: action.yml [hunk 1]
```diff
@@ -60,6 +60,22 @@ inputs:
...
+    required: false
+  FORK_REPO:
+    description: "How the 'fork repository' activity should be printed."
+    default: "🔱 Forked {FORK} from {REPO}"
+    required: false
+
+  # WIKI
+  WIKI_CREATE:
+    description: "How the 'create wiki' activity should be printed."
+    default: "📖 Created new wiki {WIKI} in {REPO}"
```
Comment (PuneetGopinath) [inline action.yml:76]: Should be `📖 Created new wiki page {WIKI} in {REPO}` instead of `📖 Created new wiki {WIKI} in {REPO}`

**Review by PuneetGopinath — CHANGES_REQUESTED** [review]

File: action.yml (hunk 1 above)
Comment (abhijoshi2k) [reply action.yml]: Done

**Comment by abhijoshi2k** [comment]
@Andre601 @PuneetGopinath 

The events added are as follows:
<br>
```
CREATE_REPO:
    description: "How the 'create new repository' activity should be printed."
    default: "📔 Created new repository {REPO}"
    required: false
```
Disable String: `create_repo`
<br>
```
FORK_REPO:
    description: "How the 'fork repository' activity should be printed."
    default: "🔱 Forked {FORK} from {REPO}"
    required: false
```
Disable String: `fork`
<br>
```
WIKI_CREATE:
    description: "How the 'create wiki' activity should be printed."
    default: "📖 Created new wiki page {WIKI} in {REPO}"
    required: false
```
Disable String: `wiki`
<br>
```
ADDED_MEMBER:
    description: "How the 'added as collaborator' activity should be printed."
    default: "🤝 Became collaborator on {REPO}"
    required: false
```
Disable String: `member`
<br>
```
REVIEW_APPROVED:
    description: "How the 'review approved' activity should be printed."
    default: "👍 Approved {ID} in {REPO}"
    required: false
CHANGES_REQUESTED:
    description: "How the 'changes requested' activity should be printed."
    default: "🔴 Requested changes in {ID} in {REPO}"
    required: false
```
Disable String: `review`
<br>
```
RELEASE:
    description: "How the 'release' activity should be printed."
    default: "✌️ Released {ID} in {REPO}"
    required: false
```
Disable String: `release`
<br>
```
STAR:
    description: "How the 'starred' activity should be printed."
    default: "⭐ Starred {REPO}"
    required: false
```
Disable String: `star`

**Comment by PuneetGopinath** [comment]
Yep, they are correct

**Review by abhijoshi2k — APPROVED** [review]

**Comment by Andre601** [comment]
I would assume the version will still be 1.4.0? Considering we didn't release that version yet would it make sense to keep.

**Comment by PuneetGopinath** [comment]
> I would assume the version will still be 1.4.0? Considering we didn't release that version yet would it make sense to keep.

Yes, v1.4.0

**Review by PuneetGopinath — APPROVED** [review]

**Review by PuneetGopinath — APPROVED** [review]

**Pull request merged**